capstone.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Electrical Engineering Portal</title>
+    <link rel="stylesheet" href="capstone.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+</head>
+
+
+
+<header>
+    <nav class="container">
+        <div class="upper-nav">
+                <a href="https://www.facebook.com/haftamudesta"><i class="fa-brands fa-4x fa-facebook"></i></a> 
+            <i class="fa-brands fa-twitter-square"></i>
+            <p>English</p>
+            <p>My page</p>
+            <p>Logout</p>
+        </div>
+        <div class="lower-nav">
+            <div class="logo">
+                <a href="capstone.html"><img src="images/electric-power-engineers1.png" alt=""></a>
+                <img src="images/Enabled.png" class="hamburger" alt="hambergur menu">
+                <p class="close">&times;</p>
+            </div>
+            <div class="information">
+                <ul>
+                    <li>
+                        <a href="about.html" class="is-active" id="information-active">About</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Program</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Join</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Sponser</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">News</a>
+                    </li>
+                </ul>
```

Comment:
- [ ] Your bottom navbar links in about.html still has a different color from original template please fix it. See screenshots 👇 
- Your design 👇 
![3](https://user-images.githubusercontent.com/67087664/181904560-da7bac14-d4e7-439e-a672-c8f06a839666.png)
- Original 👇 
![-3](https://user-images.githubusercontent.com/67087664/181904573-f8a4e30c-d4a7-4269-851f-cd5317fd184b.png)